opm/models/discretization/common/fvbasediscretization.hh

```diff
@@ -782,6 +783,7 @@ public:
      */
     void shiftIntensiveQuantityCache(unsigned numSlots = 1)
     {
+        // TODO: Consider removing this method as it i not needed when we only have one time step in the cache.
```

Comment:
But in that case it does nothing? And it is still needed for the case when we have size 2?